torchvision/ops/poolers.py

```diff
@@ -131,12 +131,10 @@ def __init__(
         canonical_level: int = 4,
     ):
         super(MultiScaleRoIAlign, self).__init__()
-        if isinstance(output_size, int):
-            output_size = (output_size, output_size)
         self.featmap_names = featmap_names
         self.sampling_ratio = sampling_ratio
-        self.output_size = tuple(output_size)
-        self.scales = None
+        self.output_size = (output_size, output_size) if isinstance(output_size, int) else tuple(output_size)
+        self.scales: Optional[Sequence[float]] = None
```

Comment:
I think JIT is not happy here! From circle CI [build](https://app.circleci.com/pipelines/github/pytorch/vision/9650/workflows/638e29aa-f199-426b-9c15-ea0106ab0170/jobs/715732).

```
def infer_scale(self, feature: Tensor, original_size: Sequence[int]) -> float:
                                                          ~~~~~~~~~~~~~ <--- HERE
        # assumption: the scale is of the form 2 ** (-k), with k integer
        size = feature.shape[-2:]
'MultiScaleRoIAlign.infer_scale' is being compiled since it was called from 'MultiScaleRoIAlign.setup_scales'
  File "/root/project/torchvision/ops/poolers.py", line 179
```

I think JIT does not support Sequence,